src/build/installation/installation.md

```diff
@@ -325,18 +354,24 @@ Although this command is used to run a function, it also builds your program.
 ## Deploy
 
 ```
-snarkos developer deploy {$PROGRAM_NAME} --private-key {$PRIVATE_KEY} --query "https://api.explorer.aleo.org/v1" --path ".build/" --broadcast "https://api.explorer.aleo.org/v1/testnet3/transaction/broadcast" --priority-fee 0
+# If you have the private key present in your .env file
+leo deploy
+
+#Otherwise
+leo deploy --private-key <your_pivate_key>
```

Comment:
leo deploy --private-key <your_private_key>

Reply:
Aren't both the commands exactly same?

Reply:
Just a spelling error "<your_pivate_key>"